examples/cos_dist.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+import arkouda as ak
+import math
+
+# dot product or two arkouda pdarrays
+def ak_dot(u, v):
+    return ak.sum(u*v)
+
+# magnitude/L_2-norm of arkouda pdarray
+def ak_mag2(u):
+    return math.sqrt(ak_dot(u,u))
+
+# cosine distance of two arkouda pdarrays
+# should function similarly to scipy.spatial.distance.cosine
+def ak_cos_dist(u, v):
+    return (1.0 - ak_dot(u,v)/(ak_mag2(u)*ak_mag2(v)))
+
+if __name__ == "__main__":
+    import argparse, sys, gc, time
+
+    parser = argparse.ArgumentParser(description="Example of cosine distance/similarity in arkouda")
+    parser.add_argument('hostname', help='Hostname of arkouda server')
```

Comment:
I recommend re-naming the hostname arg to server to match the Arkouda Python API

Reply:
I assume you wanted me to call this argument `'server'`

Reply:
Indeed! :)